lib/git.zsh

```diff
@@ -7,6 +7,14 @@ function git_prompt_info() {
   fi
 }
 
+function git_cygwin_prompt_info_quick() {
+  ref=$(git symbolic-ref HEAD) 2> /dev/null
```

Comment:
Correct; I'll amend the pull request tomorrow.




Michael

On Tue, Apr 7, 2015 at 7:05 PM, Nicolas Canceill <notifications@github.com>
wrote:

>> @@ -7,6 +7,14 @@ function git_prompt_info() {
>>    fi
>>  }
>>  
>> +function git_cygwin_prompt_info_quick() {
>> +  ref=$(git symbolic-ref HEAD) 2> /dev/null
> You probably mean:
> ```zsh
> ref=$(git symbolic-ref HEAD 2> /dev/null)
> ```
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/robbyrussell/oh-my-zsh/pull/3754/files#r27861771